redaxo/src/core/lib/console/application.php

```diff
@@ -16,6 +16,14 @@ public function __construct()
         parent::__construct('REDAXO', rex::getVersion());
     }
 
+    public function setShutdownFunction($c)
```

Comment:
Alternative Idee: In rex_console_application am Ende von `doRun` oder `doRunCommand` einen Extension Point setzen, und da als Params command, exitCode etc übergeben.
Wäre vielleicht mehr redaxo-like, oder?
Und wir bräuchte die event-dispatcher komponente von symfony nicht.

Reply:
finde ich eine gute idee. ich hatte ohnehin versucht die symfony implementierung hier zu verstecken.